cidc_api/emails.py

```diff
@@ -71,13 +73,23 @@ def new_user_registration(email: str) -> dict:
 
 
 @sendable
-def new_upload_alert(upload: Union[AssayUploads, ManifestUploads]) -> dict:
+def new_upload_alert(
+    upload: Union[AssayUploads, ManifestUploads], full_metadata: dict
+) -> dict:
     """Alert the CIDC administrators that an upload succeeded."""
 
+    patch = (
+        upload.assay_patch if hasattr(upload, "assay_patch") else upload.metadata_patch
+    )
+
     upload_type = (
         upload.assay_type if hasattr(upload, "assay_type") else upload.manifest_type
     )
 
+    pipeline_configs = generate_analysis_configs_from_upload_patch(
+        full_metadata, patch, upload_type, GOOGLE_DATA_BUCKET
```

Comment:
Is the idea that we will come back in a future PR and introduce the concept of a separate biofx data bucket?